### ISSUE TITLE
Serenity with gecko driver does not allow AXE core accessibility to run

### ISSUE THREAD

**Issue by maheshdaniin** [serenity-bdd/serenity-core#3224]
This is in relation to issue raised with AXE core
https://github.com/dequelabs/axe-core-maven-html/issues/315

when tried to run accessibility check against web page open in geckodriver via serenity it fails with below error 
#Custom web driver details
webdriver.driver=firefox

1690471863433 RemoteAgent INFO Perform WebSocket upgrade for incoming connection from 127.0.0.1:55526
JavaScript error: chrome://remote/content/server/WebSocketHandshake.sys.mjs, line 168: Error: The handshake request has incorrect Origin header http://127.0.0.1:49664/
16:31:03.436 [AsyncHttpClient-1-4] DEBUG org.asynchttpclient.netty.handler.WebSocketHandler -

Request DefaultFullHttpRequest(decodeResult: success, version: HTTP/1.1, content: EmptyByteBufBE)
GET /devtools/browser/9d1b75b0-620b-4e4c-90e2-6a1f1a23a9d9 HTTP/1.1
upgrade: websocket
connection: upgrade
sec-websocket-key: Rrq44R34IYTbr1YwLJ7f3Q==
sec-websocket-version: 13
origin: http://127.0.0.1:49664/
host: 127.0.0.1:49664
accept: /
user-agent: AHC/2.1

Response DefaultHttpResponse(decodeResult: success, version: HTTP/1.1)
HTTP/1.1 400 Bad Request
Server: httpd.js
Content-Type: text/plain
content-length: 72

Jul 27, 2023 4:31:03 PM org.openqa.selenium.remote.http.WebSocket$Listener onError
WARNING: Invalid Status code=400 text=Bad Request
java.io.IOException: Invalid Status code=400 text=Bad Request

when checked with AXE core team 
they have following comment 

The default geckodriver with pure Selenium works as intended. It appears that Serenity does not like we open a blank new tab as part of the Selenium driver (outside of Senerity) as well as not allowing enough time to close the page before going onto the next definition.

Please refer to the the issue for complete history of this discussion with them

Please could you advise how do we go about this and get accessibility run using gecko driver with serenity?
